addons/binding/org.openhab.binding.network/src/main/java/org/openhab/binding/network/service/NetworkUtils.java

```diff
@@ -167,4 +167,25 @@ public static boolean nativePing(String hostname, int port, int timeout)
         return exitValue == 0;
     }
 
+    public static boolean arping(String hostname, int timeout, String networkInterface)
+            throws InvalidConfigurationException, IOException, InterruptedException {
+        Process proc;
+        if (SystemUtils.IS_OS_UNIX) {
+            proc = new ProcessBuilder("arping", "-I", networkInterface, "-w", String.valueOf(timeout / 1000), "-c", "1",
```

Comment:
What's the benefit of the testing command instead of just executing the regular command? And why set this.useArping to false? Do you suggest a fallback to regular ping? I don't know if that's wise.